webpack.config.js

```diff
@@ -0,0 +1,48 @@
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+    mode: 'development',
+    entry: './src/index.js',
+    output: {
+        path: path.resolve(__dirname,'dist'),
+        filename: 'bundle.js',
+        clean: true
+    },
+    devServer:{
+        static: './dist',
+    },
+    module:{
+        rules:[
+            {
+                test: /\.css$/i,
+                use: [MiniCssExtractPlugin.loader, "css-loader"],
+              },
+            // {
+            //     test:/\.css$/i,
+            //     use:['style-loader', 'css-loader']
+            // },
+            {
+                test: /\.(png|svg|jpg|jpeg|gif)$/i,
+                type: 'asset/resource',
+            },
+            {
+                test: /\.(woff|woff2|eot|ttf|otf)$/i,
+                type: 'asset/resource',
+              },
+            {
+                test:/\.html&/i, 
+                use: 'html-loader'
+            }
+        ]
+    },
+    plugins: [
+        new HtmlWebpackPlugin({
+            template: 'src/index.html',
+            title: 'Calendar',
+            filename:'calendar.html'
```

Comment:
Please remove filename. The entry point for the site/application is always index.html

Reply:
Done